app/main.py

```diff
@@ -1,3 +1,10 @@
 def cache(func):
-    # Write your code here
-    pass
+    temp_dict = {}
+    def inner(*args):
+        if args in temp_dict:
+            print(f"Getting from cache")
```

Comment:
Please, fix flake8 errors: string shouldn't be f-string if it doesn't contain placeholders

Reply:
Done